content/en/docs/Writing policies/preconditions.md

```diff
@@ -15,12 +15,124 @@ For `validate` rules, the use of `patterns` is often preferable since conditiona
 
 When specifying a JMESPath expression in a `preconditions` statement which contains a special character (ex. `/` in the case of Kubernetes annotations), double quote the annotation as a literal string. Escape the double quotes with a backslash character (`\`).
 
-```yaml
+```sh
 {{request.object.spec.template.metadata.annotations.\"foo.k8s.corp.net/bar\"}}
 ```
 
 You may specify multiple statements in the `preconditions` field. The use of multiple `preconditions` statements function as a logical AND statement.
 
+## Any and All Statements
+
+You may further control how `preconditions` are evaluated by nesting the expressions under `any` and/or `all` statements. This gives you further power in building more precise logic for how the rule is triggered. Either or both may be used simultaneously in the same rule with multiple `any` statements also being possible. For each `any`/`all` statement, each block must overall evaluate to TRUE for the precondition to be processed. If any of the `any` / `all` statement blocks does not evaluate to TRUE, `preconditions` will not be satisfied and thus the rule will not be applicable.
+
+{{% alert title="Note" color="info" %}}
+`any` and `all` statements are available starting in Kyverno v1.3.4 but are optional. Users can continue to write multiple statements inside the `preconditions` field and have them evaluated as a logical AND statement.
+{{% /alert %}}
```

Comment:
```suggestion
```

Reply:
So your suggestion here is to keep the text but don't put it in an information note? Any particular reason you feel that way?

Reply:
Markdown is unable to process this syntax.

Reply:
The syntax is not for markdown, it's shortcode for the Docsy Hugo theme. It's what produces this effect:

![image](https://user-images.githubusercontent.com/29008123/113894763-51e85000-9796-11eb-9c62-882d970dd03c.png)

Reply:
aah, got it. my bad.